test/e2e/cluster_logging_gcl_utils.go

```diff
@@ -35,8 +35,8 @@ const (
 	// quota limit exceeded. So we retry for some time in case the problem will go away.
 	// Quota is enforced every 100 seconds, so we have to wait for more than
 	// that to reliably get the next portion.
-	queryGclRetryDelay   = 10 * time.Second
-	queryGclRetryTimeout = 200 * time.Second
```

Comment:
3 times. Imagine we're out of quotas:

It requests API, fails, waits for 100 sec. Then tries again (with different quota already!), fails, waits for 100 sec. Tries again, fails, waits for 100 sec and then exists.

Then it exists this functions and waits for another 100 sec. ~400sec in total, out of 600.

In total, it will try in ingest logs 6 times before failing the test.